CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 <!-- ## [Unreleased] -->
 
+## [2.12.1] - 2020-03-18
+
+- Adjusted Figma metadata for variants. Adjusted description of one variant.
```

Comment:
You'll want to add a link to the pull request where this was done. You can see the other entries for an example and formatting

Reply:
Thanks Dan, fixed!